libs/sdk-backend-spi/src/workspace/measures/index.ts

```diff
@@ -52,4 +52,19 @@ export interface IWorkspaceMeasuresService {
      * @returns promise of references
      */
     getMeasureReferencingObjects(measureRef: ObjRef): Promise<IMeasureReferencing>;
+
+    /**
+     * Get all metadata objects for given measure references.
+     *
+     * @remarks
+     * If the array of the given references is too large, the function must load all the measures
```

Comment:
I think we should hide this in the implementation + I think it's not necessary. If the array of the given references is too large, we can divide it to multiple requests, run them in parallel and then return just all the results.

Reply:
Discussed offline, we decided that this further optimization would not likely help in common use cases.